ownCloud/Key Commands/KeyCommands.swift

```diff
@@ -1016,13 +1042,90 @@ extension DisplayHostViewController {
 			shortcuts.append(previousObjectCommand)
 		}
 
+		if let core = core, let currentViewController = self.viewControllers?.first as? DisplayViewController, let item = currentViewController.item {
```

Comment:
@hosy try avoiding long lines for readability

Reply:
Fixed!